framework/base/Module.php

```diff
@@ -729,4 +729,17 @@ public function get($id, $throwException = true)
         }
         return $component;
     }
+
+    /**
+     * @inheritdoc
+     * @param string $id
+     * @param bool $checkInstance
+     * @return bool
+     * Since version 2.0.13, if a component isn't defined in the module, it will be looked up in the parent module.
+     * The parent module may be the application.
```

Comment:
You should remove `@return` and `@param` annotations - they will be inherited from parent class.

Reply:
done.